pkg/acl/access_control_list_middleware_test.go

```diff
@@ -48,22 +47,34 @@ func Test_AccessControlListMiddleware_UserHasNoAccess(t *testing.T) {
 	}
 
 	tests := []struct {
-		name string
-		arg  *acl.AccessControlListConfig
-		want *errors.ServiceError
+		name           string
+		arg            *acl.AccessControlListConfig
+		wantErr        bool
+		wantHttpStatus int
 	}{
 		{
 			name: "returns 403 Forbidden response when user is not allowed to access service",
 			arg: &acl.AccessControlListConfig{
 				EnableDenyList: true,
 				DenyList:       acl.DeniedUsers{"username"},
 			},
+			wantErr:        true,
+			wantHttpStatus: http.StatusForbidden,
+		},
+		{
+			name: "returns 200 status if denyList is disabled",
+			arg: &acl.AccessControlListConfig{
+				EnableDenyList: false,
+			},
+			wantErr:        false,
+			wantHttpStatus: http.StatusOK,
 		},
```

Comment:
We are missing a test case when user is allowed to access the service and deny list is enabled